.github/workflows/pr.yml

```diff
@@ -7,7 +7,7 @@ name: Build pr
 
 on:
   pull_request:
-    branches: [main, release/**, next/**, feat/add-jest-tests]
+    branches: ['**']
```

Comment:
Hmmm?

Reply:
This was to enable testing the sub-branches from this one. Reverting to main, release and next only.